app/components/avo/views/resource_edit_component.rb

```diff
@@ -71,13 +97,19 @@ def is_edit?
   end
 
   def form_method
-    return :put if is_edit?
+    return :put if is_edit? && params[:controller] != "avo/bulk_update"
```

Comment:
I think we should use the put as well for bulk updates, any argument why we should use post?